### PR TITLE
Copies zips from local staging to m2

### PR DESCRIPTION
**Pull request by owaiskazi19**
### Description
Coming from https://github.com/opensearch-project/job-scheduler/pull/581#issuecomment-1935250785
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -76,4 +76,5 @@ cp ./build/distributions/*.zip $OUTPUT/plugins
 ./gradlew publishAllPublicationsToStagingRepository -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
 ./gradlew publishPluginZipPublicationToZipStagingRepository -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
 mkdir -p $OUTPUT/maven/org/opensearch
-cp -r ./build/local-staging-repo/org/opensearch/. $OUTPUT/maven/org/opensearch
\ No newline at end of file
+cp -r ./build/local-staging-repo/org/opensearch/. $OUTPUT/maven/org/opensearch
+cp -r ./build/local-staging-repo/org/opensearch/plugin/opensearch-job-scheduler ~/.m2/repository/org/opensearch/plugin/
```
Comment (prudhvigodithi) [inline scripts/build.sh:80]: @owaiskazi19 please add a comment linking the skills repo issue.

Comment (peterzhuamazon) [reply scripts/build.sh:81]: ^^

Comment (owaiskazi19) [reply scripts/build.sh:81]: Done

**Comment by prudhvigodithi** [comment]
I see this works, skills does not complain job-scheduler (as seen in [past](https://github.com/opensearch-project/skills/issues/178#issuecomment-1931244149)) as it copied to local maven.
```
* What went wrong:
A problem occurred evaluating root project 'opensearch-skills'.
> Could not resolve all files for configuration ':zipArchive'.
   > Could not find org.opensearch.plugin:opensearch-ml-plugin:2.12.0.0.
     Searched in the following locations:
       - file:/home/ci-runner/.m2/repository/org/opensearch/plugin/opensearch-ml-plugin/2.12.0.0/opensearch-ml-plugin-2.12.0.0.pom
       - https://repo.maven.apache.org/maven2/org/opensearch/plugin/opensearch-ml-plugin/2.12.0.0/opensearch-ml-plugin-2.12.0.0.pom
       - https://plugins.gradle.org/m2/org/opensearch/plugin/opensearch-ml-plugin/2.12.0.0/opensearch-ml-plugin-2.12.0.0.pom
       - https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/opensearch-ml-plugin/2.12.0.0/opensearch-ml-plugin-2.12.0.0.pom
     Required by:
         project :
   > Could not find org.opensearch.plugin:opensearch-anomaly-detection:2.12.0.0.
     Searched in the following locations:
       - file:/home/ci-runner/.m2/repository/org/opensearch/plugin/opensearch-anomaly-detection/2.12.0.0/opensearch-anomaly-detection-2.12.0.0.pom
       - https://repo.maven.apache.org/maven2/org/opensearch/plugin/opensearch-anomaly-detection/2.12.0.0/opensearch-anomaly-detection-2.12.0.0.pom
       - https://plugins.gradle.org/m2/org/opensearch/plugin/opensearch-anomaly-detection/2.12.0.0/opensearch-anomaly-detection-2.12.0.0.pom
       - https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/opensearch-anomaly-detection/2.12.0.0/opensearch-anomaly-detection-2.12.0.0.pom
     Required by:
         project :
   > Could not find org.opensearch.plugin:opensearch-sql-plugin:2.12.0.0.
     Searched in the following locations:
       - file:/home/ci-runner/.m2/repository/org/opensearch/plugin/opensearch-sql-plugin/2.12.0.0/opensearch-sql-plugin-2.12.0.0.pom
       - https://repo.maven.apache.org/maven2/org/opensearch/plugin/opensearch-sql-plugin/2.12.0.0/opensearch-sql-plugin-2.12.0.0.pom
       - https://plugins.gradle.org/m2/org/opensearch/plugin/opensearch-sql-plugin/2.12.0.0/opensearch-sql-plugin-2.12.0.0.pom
       - https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/opensearch-sql-plugin/2.12.0.0/opensearch-sql-plugin-2.12.0.0.pom
     Required by:
         project :
   > Could not find org.opensearch.plugin:opensearch-knn:2.12.0.0.
     Searched in the following locations:
       - file:/home/ci-runner/.m2/repository/org/opensearch/plugin/opensearch-knn/2.12.0.0/opensearch-knn-2.12.0.0.pom
       - https://repo.maven.apache.org/maven2/org/opensearch/plugin/opensearch-knn/2.12.0.0/opensearch-knn-2.12.0.0.pom
       - https://plugins.gradle.org/m2/org/opensearch/plugin/opensearch-knn/2.12.0.0/opensearch-knn-2.12.0.0.pom
       - https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/opensearch-knn/2.12.0.0/opensearch-knn-2.12.0.0.pom
     Required by:
         project :
   > Could not find org.opensearch.plugin:neural-search:2.12.0.0.
     Searched in the following locations:
       - file:/home/ci-runner/.m2/repository/org/opensearch/plugin/neural-search/2.12.0.0/neural-search-2.12.0.0.pom
       - https://repo.maven.apache.org/maven2/org/opensearch/plugin/neural-search/2.12.0.0/neural-search-2.12.0.0.pom
       - https://plugins.gradle.org/m2/org/opensearch/plugin/neural-search/2.12.0.0/neural-search-2.12.0.0.pom
       - https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/neural-search/2.12.0.0/neural-search-2.12.0.0.pom
     Required by:
         project :
   > Could not find org.opensearch.plugin:alerting:2.12.0.0.
     Searched in the following locations:
       - file:/home/ci-runner/.m2/repository/org/opensearch/plugin/alerting/2.12.0.0/alerting-2.12.0.0.pom
       - https://repo.maven.apache.org/maven2/org/opensearch/plugin/alerting/2.12.0.0/alerting-2.12.0.0.pom
       - https://plugins.gradle.org/m2/org/opensearch/plugin/alerting/2.12.0.0/alerting-2.12.0.0.pom
       - https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/alerting/2.12.0.0/alerting-2.12.0.0.pom
     Required by:
         project :

```

**Review by prudhvigodithi — APPROVED** [review]

**Comment by prudhvigodithi** [comment]
@peterzhuamazon do you see any issues with windows?

**Comment by prudhvigodithi** [comment]
Instead of copying, I have a PR open which publishes the zips to maven local in gradle way https://github.com/opensearch-project/job-scheduler/pull/584.
@peterzhuamazon @bbarani

**Comment by owaiskazi19** [comment]
@prudhvigodithi can we close this then?

**Comment by prudhvigodithi** [comment]
Yes @owaiskazi19 closing this in favor of https://github.com/opensearch-project/job-scheduler/pull/584. Thanks

**Pull request closed without merging**